controllers/datadogagent/feature/helmcheck/configmap.go

```diff
@@ -0,0 +1,60 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package helmcheck
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/DataDog/datadog-operator/controllers/datadogagent/object/configmap"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (f *helmCheckFeature) buildHelmCheckConfigMap() (*corev1.ConfigMap, error) {
+	if f.customConfig != nil && f.customConfig.ConfigMap != nil {
+		return nil, nil
+	}
+	if f.customConfig != nil && f.customConfig.ConfigData != nil {
+		return configmap.BuildConfigMapConfigData(f.owner.GetNamespace(), f.customConfig.ConfigData, f.configMapName, helmCheckConfFileName)
+	}
+
+	configMap := buildDefaultConfigMap(f.owner.GetNamespace(), f.configMapName, helmCheckConfig(f.runInClusterChecksRunner, f.collectEvents, f.valuesAsTags))
+	return configMap, nil
+}
+
+func buildDefaultConfigMap(namespace, cmName string, content string) *corev1.ConfigMap {
+	configMap := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      cmName,
+			Namespace: namespace,
+		},
+		Data: map[string]string{
+			helmCheckConfFileName: content,
+		},
+	}
+	return configMap
+}
+
+func helmCheckConfig(clusterCheck bool, collectEvents bool, valuesAsTags map[string]string) string {
```

Comment:
Can the helm check be run in the node agent (running cluster checks on node agents) or can it only be run in the DCA/CCR? If it's the latter, can we add a note similar to [what we have for ksm](https://github.com/DataDog/datadog-operator/blob/6f5b5c50b330122b8c1f49320ec8d2097f422b75/controllers/datadogagent/feature/kubernetesstatecore/configmap.go#L42-L47)

Reply:
Yup, it can only run in the DCA/CCR. I'll add the note